### DISCUSSION TITLE
Time for 0.9.8?

### DISCUSSION THREAD

**Discussion by liu-song-6** [dynup/kpatch#1333] [Q&A] [answered]
0.9.7 was released 5 months ago. Shall we release 0.9.8?

**Comment by jpoimboe** [comment, marked as answer]
Sure, it's a good a time as any other ;-)  We tend to keep the master branch pretty stable, if we were more organized we would do more regular releases.

**Reply by liu-song-6** [reply]
So... shall we cut 0.9.8 this week? Btw, we are running out of numbers before 1.0.0. :)

**Reply by joe-lawrence** [reply]
Yes, we are converging on 1.0.0 soon!

As for 0.9.8, how about this changelog,
```
Changes since v0.9.7:
- Clang fix ups from Pete Swain
- Support for gcc-12
- Support for linux kernel 5.19
- Added RHEL-8.7 and 9.1 integration tests
- Fixed __UNIQUE_ID() variable correlation
- Improved handling of unsupported static calls
```
Anything else catch your eye that we should highlight?

**Reply by jpoimboe** [reply]
> Btw, we are running out of numbers before 1.0.0. :)

For version x.y.z we usually only bump the "y" for some backwards-incompatible change.  There'd be no harm in bumping the "z" to double digits if we needed to.

That said, I wouldn't be opposed to a 1.0.0 at some point, kpatch has definitely proven to be production-worthy for several years now.

@joe-lawrence the changelog looks good, thanks for organizing the release.

**Reply by joe-lawrence** [reply]
Done.  https://github.com/dynup/kpatch/releases/tag/v0.9.8
